scripts/add_sol_pool.sh

```
@@ -0,0 +1,217 @@
+KONG_BACKEND=$(dfx canister id ${NETWORK_FLAG} kong_backend)
+
+# Token configurations
+SOL_AMOUNT=10_000_000  # 0.01 SOL (9 decimals)
+SOL_AMOUNT=${SOL_AMOUNT//_/}  # remove underscore
+SOL_CHAIN="SOL"
+SOL_ADDRESS="11111111111111111111111111111111"  # Native SOL
+
+# ksUSDT configuration
+KSUSDT_SYMBOL="ksUSDT"
+KSUSDT_CHAIN="IC"
+KSUSDT_LEDGER=$(dfx canister id ${NETWORK_FLAG} $(echo ${KSUSDT_SYMBOL} | tr '[:upper:]' '[:lower:]')_ledger)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)